modules/tensor_mechanics/unit/src/RankTwoTensorTest.C

```diff
@@ -540,9 +540,10 @@ TEST_F(RankTwoTensorTest, ADConversion)
   generic_reg = MetaPhysicL::raw_value(generic_ad);
 }
 
-TEST_F(RankTwoTensorTest, mixedProductIjJklm)
+TEST_F(RankTwoTensorTest, mixedProductIMMJKL)
```

Comment:
This test passes on new compilers but gives wrong results on min clang.On every other compiler (including gcc min) the template instantiation `mixedProduct<i, m, m, j, k, l>` works fine. All other tested instantiations of this same template work on all compilers.

What the heck?